ValheimExportHelper/ValheimExportHelper.csproj

```diff
@@ -49,6 +48,15 @@
       <HintPath>..\..\Downloads\AssetRipper-0.2.2.0\Bins\AssetRipper\Release\AssetRipper.Library.dll</HintPath>
       <Private>False</Private>
     </Reference>
+    <Reference Include="UnityEditor">
+      <HintPath>..\..\..\..\..\Program Files\Unity\Hub\Editor\2020.3.33f1\Editor\Data\Managed\UnityEditor.dll</HintPath>
+    </Reference>
+    <Reference Include="UnityEngine">
+      <HintPath>..\..\..\..\..\Program Files\Unity\Hub\Editor\2020.3.33f1\Editor\Data\Managed\UnityEngine.dll</HintPath>
+    </Reference>
+    <Reference Include="UnityEngine.AssetBundleModule">
+      <HintPath>..\..\..\..\..\Program Files\Unity\Hub\Editor\2020.3.33f1\Editor\Data\Managed\UnityEngine\UnityEngine.AssetBundleModule.dll</HintPath>
+    </Reference>

```

Comment:
What are these needed for?

Reply:
The Asset Bundle creation script. 

I wasn't really sure if the monoscripts are considered in compilation or if they are merely repackaged as a resource 

Can be dropped if scripts dont actually compile into anything

Reply:
Nah just right click the `.cs` file that's in Resource -> Properties -> Set Build Action to None.